server/embed/etcd.go

```diff
@@ -418,7 +422,11 @@ func stopServers(ctx context.Context, ss *servers) {
 	}
 }
 
-func (e *Etcd) Err() <-chan error { return e.errc }
+// Err - return channel used to report errors during etcd run/shutdown.
+// Since etcd 3.5 the channel is being closed when the etcd is over.
```

Comment:
Let's also add this behavior change to the changelog.

Reply:
Done.